_layouts/tutorial_hands_on.html

```diff
@@ -224,6 +224,58 @@ <h1 id="bibliography">References</h1>
                     {% bibliography --cited %}
                     {% endif %}
 
+                    <h1>Citing this Tutorial</h1>
+                    <p>
+                        <ol>
+                            <li id="citation-text">
+                            {%- include _includes/contributor-author-list-comma.html contributors=page.contributors -%}, {{ page.last_modified_at | date: "%Y" }} <b>{{ page.title }} (Galaxy Training Materials)</b>. <a href="{{ site.baseurl }}{{page.url}}">{{ site.baseurl }}{{page.url}}</a> Online; accessed TODAY
+                            </li>
+                            <li>
+                            Batut et al., 2018 <b>Community-Driven Data Analysis Training for Biology</b> Cell Systems <a href="https://doi.org/10.1016%2Fj.cels.2018.05.012">10.1016/j.cels.2018.05.012</a>
```

Comment:
Is it possible to use things as the citations in the tutorial?

Reply:
No, it is not, unfortunately. We cannot easily call that code from within a template, or we would have to modifying the content generation pipeline to generate an extra bib file for every single tutorial, and it would need to be early enough in the pipeline that scholar could discover it, it really would not be easy unfortunately.